apps/etterlatte-behandling/src/main/kotlin/behandling/brevoppsett/Brevoppsett.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.etterlatte.behandling.brevoppsett
+
+import no.nav.etterlatte.behandling.domain.Behandling
+import no.nav.etterlatte.libs.common.feilhaandtering.IkkeTillattException
+import no.nav.etterlatte.libs.common.feilhaandtering.UgyldigForespoerselException
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import java.time.LocalDate
+import java.time.YearMonth
+import java.util.UUID
+
+data class Brevoppsett(
+    val behandlingId: UUID,
+    val brevtype: Brevtype,
+    val etterbetaling: Etterbetaling?,
+    val aldersgruppe: Aldersgruppe?,
+    val kilde: Grunnlagsopplysning.Kilde,
+)
+
+data class Etterbetaling(
+    val fom: YearMonth,
+    val tom: YearMonth,
+) {
+    init {
+        if (fom > tom) {
+            throw BrevoppsettException.EtterbetalingFomErEtterTom(fom, tom)
+        }
+        if (tom > YearMonth.now()) {
+            throw BrevoppsettException.EtterbetalingTomErFramITid(tom)
+        }
+    }
+
+    companion object {
+        fun fra(
+            datoFom: LocalDate?,
+            datoTom: LocalDate?,
+        ): Etterbetaling {
+            if (datoFom == null || datoTom == null) {
+                throw BrevoppsettException.EtterbetalingManglerDato()
+            }
+            return Etterbetaling(YearMonth.from(datoFom), YearMonth.from(datoTom))
+        }
+    }
+}
+
+enum class Brevtype {
```

Comment:
Burde dette hete `Region` / `Omraade` e.l. noe i den gaten? Aner ikke hva det heter i andre systemer. Eventuelt forenkle til `utland: boolean` siden dette (så vidt jeg vet) alltid kun har to utfall? 

Føler hvert fall at "brevtype" fort vekk kan forveksles med andre ting. 🤔

Reply:
Denne fjerner vi nå som utlandstilknytning ligger på behandling. Da er hypotesen at den kan utledes derfra.

Reply:
Fjernet denne i siste commit